decision-making-processes.md

```diff
@@ -1,35 +1,157 @@
-Here is the start of a description of the decision making process for the variety of decision types. Deciding how to decide is a work in progress that is being refined. 
+# Abstract 
+This document defines the formal governance process for Solid, and records how this has changed over time. Currently, governance is based around Solid Team who consult the Solid Panel for their opinion via a vote on occasion. The Solid Team has broad authority, which they seek to exercise as rarely as possible. The Solid Leader (who is in the Solid Team) needs to approve any decision making processes for it to be legitimate and the Solid Leader can veto decisions at any point. 
 
-The Solid Leader needs to give final approval the decision making process for each category of decision including who can decide on what to the decision to hold legitimacy. 
+# Scope
+The scope of Solid governance includes but it not limited to: 
+* Changes to the Solid specification. Decisions around the Solid Specification changes need to have a procedure around them to encourage conversation to find accepted legitimate solutions to more complex decisions. 
+* How to start a [Solid Project](https://github.com/orgs/solid/projects) i.e. a GitHub project in the Solid Github account? 
+* What to include on the [Solid roadmap](https://github.com/solid/information/blob/master/solid-roadmap.md)? 
+* Which communication channels to use for Solid conversations? 
+* What to include in the Solid test suite?
 
-Some decisions can be made purely by the Solid Leader; however, there are many decisions at a lower category of important for which it would be useful to have a legitimate process that could always be blocked by the Solid leader. The operations of legitimate process could be [...?]
+# [The Solid Team]
+The Solid Team is a 5 person trusted group who manage Solid, which currently consists of: 
```

Comment:
"manage" has many meanings; what do we mean?

Reply:
Management scope is covered in the section #Mandate

Reply:
Best to link that word to the relevant section then